CMakeLists.txt

```diff
@@ -1,10 +1,11 @@
 # CMakeLists.txt has to be located in the project folder and cmake has to be
 # executed from 'project/build' with 'cmake ../'.
 cmake_minimum_required(VERSION 3.1)
+project(dummy_project VERSION 0.1)
```

Comment:
I would add the DESCRIPTION field here as well

Reply:
Agreed.

Reply:
Done. Hacky, because the info was in config_manifest. But I really don't want to get into a refactoring of this shell-based template "system".